Lesson_07/homework_7.py

```diff
@@ -0,0 +1,217 @@
+# task 1
+""" Задача - надрукувати табличку множення на задане число, але
+лише до максимального значення для добутку - 25.
+Код майже готовий, треба знайти помилки та виправити\доповнити.
+"""
+from typing import List
+
+
+def multiplication_table(number):
+    # Initialize the appropriate variable
+    multiplier = 1
+
+    # Complete the while loop condition.
+    while multiplier <= number:
+        result = number * multiplier
+
+        if result > 25:
+            # Enter the action to take if the result is greater than 25
+            pass
+        print(str(number) + "x" + str(multiplier) + "=" + str(result))
+
+        # Increment the appropriate variable
+        multiplier += 1
+
+
+multiplication_table(3)
+
+# task 2
+"""  Написати функцію, яка обчислює суму двох чисел.
+"""
+
+
+def sum_numbers(a, b):
+    return a + b
+
+
+result = sum_numbers(3, 5)
+print(result)
+
+# task 3
+"""  Написати функцію, яка розрахує середнє арифметичне списку чисел.
+"""
+
+
+def average(numbers):
+    if not numbers:
+        return 0
+    total = sum(numbers)
```

Comment:
It better and more clear no to create a big amount of unnecessary local variables. It is enough really to code like - 
return sum(numbers) / len(numbers)